lte/gateway/c/core/oai/lib/hashtable/hashtable_uint64.c

```diff
@@ -708,6 +708,95 @@ hashtable_rc_t hashtable_uint64_ts_insert(
   return HASH_TABLE_OK;
 }
 
+//------------------------------------------------------------------------------
+/*
+   To free_wrapper an element from the hash table, we just search for it in the
+   linked list for that hash value, and free_wrapper it if it is found. If it
+   was not found, it is an error and -1 is returned.
+*/
+hashtable_rc_t hashtable_uint64_free(
```

Comment:
this function is the same as `hashtable_uint64_remove`, please remove this and reuse that function.

Reply:
removed hashtable_uint64_free function and replaced caller APIs with hashtable_uint64_remove